frontend/archive/src/main/java/org/pytorch/serve/archive/utils/ArchiveUtils.java

```diff
@@ -47,7 +48,7 @@ public static <T> T readFile(File file, Class<T> type)
 
     public static <T> T readYamlFile(File file, Class<T> type)
             throws InvalidModelException, IOException {
-        Yaml yaml = new Yaml(new Constructor(type));
+        Yaml yaml = new Yaml(new Constructor(type, new LoaderOptions()));
```

Comment:
With 2.1 version of snakeyaml, constructor definition has changed - https://bitbucket.org/snakeyaml/snakeyaml/src/1631d43d63fe468d8232a80f47608e95615e5b29/src/main/java/org/yaml/snakeyaml/constructor/Constructor.java#lines-59